optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/util/PlannerTestUtils.java

```diff
@@ -85,6 +89,21 @@ public Score calculateScore(Solution solution) {
 
     }
 
+    public static String readResourceToString(Class clazz, String resource) throws IOException {
+        InputStream resourceInputStream = clazz.getResourceAsStream(resource);
+        String resourceContent = "";
+        String buffer;
+        BufferedReader br = new BufferedReader(new InputStreamReader(resourceInputStream));
```

Comment:
Never create a reader without specifying that the encoding is UTF-8, or we get platform dependent code (= bugs on windows that default to the encoding "windows-1252"). findbugs finds these bugs for us.